src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -311,6 +311,7 @@ export class PatientViewPageStore {
 
     public internalClient: CBioPortalAPIInternal;
 
+    @observable public activeLocus: string | undefined;
```

Comment:
Another alternative is to create an explicit `GenomicOverviewStore` in `PatientViewPage` and move this observable there instead of keeping it in `PatientViewPageStore`.

Reply:
I'm confused, it seems like you did this below, no?  i mean make a store.

Reply:
We initialize a new `DefaultGenomicOverviewStore` within the `GenomicOverview` component only if `prop.store` is not defined. However, we are passing the entire `PatientViewPageStore` instance to the component in [PatientViewPageTabs](https://github.com/cBioPortal/cbioportal-frontend/pull/4336/files#diff-7015cbed8f5eb4c9f9fa577da0b6b436038f0f6ea4609bf4a9e64a9a8624ca3fR191). 

So, the actual runtime instance is an instance of `PatientViewPageStore`, not an instance of a `DefaultGenomicOverviewStore`. Alternatively, we can actually instantiate a new `DefaultGenomicOverviewStore` in patient view page instead of passing `PatientViewPageStore` to the component. Probably, not a very important implementation detail though.